app/src/main/res/values/strings.xml

```diff
@@ -673,41 +673,41 @@
     <string name="onboard">Onboard</string>
     <string name="do_you_want_to_uninstall_and_install">Do you want to uninstall and install the app again?</string>
-    <string name="signature_mismatch_for_system_apps">Could not install app because the installed app is a system app with different signature.</string>
+    <string name="signature_mismatch_for_system_apps">Could not install app because the installed one is a system app with different signature.</string>
     <string name="app_data_will_be_lost">The existing data will be lost.</string>
-    <string name="install_without_data_loss">If you have disabled signature verification, you can use the <b>only install</b> option to install the app without losing data.</string>
+    <string name="install_without_data_loss">If you have turned off signature verification, you can use the <b>Only install</b> option to install the app without losing data.</string>
     <string name="only_install">Only Install</string>
     <string name="sys_config">System Config</string>
     <string name="scanner">Scanner</string>
     <string name="no_libs">No libraries</string>
     <string name="lib_details">Library Details</string>
     <string name="tap_to_see_details">Tap to see details</string>
-    <string name="view_missing_signatures">Tap to view signatures that are not yet present in App Manager\'s database of libraries. You can also send them to me if you want.</string>
+    <string name="view_missing_signatures">Tap to view or send in signatures not yet present in App Manager\'s database of libraries.</string>
     <string name="copy">Copy</string>
     <string name="filter_running_apps">Running apps</string>
     <string name="am_crashed">AM is just crashed</string>
     <string name="tap_to_submit_crash_report">Tap to submit the crash report.</string>
     <string name="send_crash_report">Send crash report</string>
-    <string name="select_apk">Select apk</string>
+    <string name="select_apk">Select APK</string>
     <string name="conversion_failed">Conversion failed.</string>
     <string name="in_progress">In progress</string>
     <string name="un_apkm">UnApkm</string>
     <string name="input_profile_name">Profile name</string>
-    <string name="input_profile_name_description">Profile name should not contain any space.</string>
+    <string name="input_profile_name_description">The profile name can not contain any spaces.</string>
     <string name="new_profile">New profile</string>
     <string name="duplicate">Duplicate</string>
     <string name="failed_to_duplicate_profile">Could not duplicate profile</string>
     <string name="pref_show_users_in_installer">Show users in installer</string>
-    <string name="pref_show_users_in_installer_msg">Display a list of users to select before installing an external app. Requires root or ADB</string>
+    <string name="pref_show_users_in_installer_msg">Display a list of users to select before installing an external app. Requires root or ADB.</string>
```

Comment:
Again, preference items do not have a full-stop at the end. If you're going to add a full-stop to one item, you have to add them to all of them.

Reply:
There is one for the first sentence, so it follows it should be there in the second one. Will add for all.

Reply:
Thanks.